### PR TITLE
Add scRNA intro and celltyping slides

### PR DESCRIPTION
**Pull request by jashapiro**
Here I am adding the slides for the intro to single-cell with R and the cell-typing slides. I also renamed the welcome slides for ordering reasons, and added links to all three to the schedule.

### CODE REVIEW

File: workshop/SCHEDULE.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ nav_title: Schedule
 | Time      | Topic                                                            |
 |------------------|------------------------------------------------------|
 | **Day 1** | **2023-01-31**                                                   |
-| 9:00 AM   | Welcome, Introductions and Getting Started                       |
-| 10:00 AM  | Introduction/Refresher to Single Cell Analysis with R and Bioconductor           |
+| 9:00 AM   | Welcome, Introductions and Getting Started   <br> [Workshop Introduction slides (PDF)](../slides/2023-01-31_a_Workshop_Intro.pdf)  |
+| 10:00 AM  | Introduction/Refresher to Single-cell Analysis with R and Bioconductor  <br> [Single-cell with R slides (PDF)](../slides/2023-01-31_b_scRNAseq_intro.pdf)    |
```
Comment (sjspielman) [inline workshop/SCHEDULE.md:12]:
```suggestion
| 10:00 AM  | Introduction/Refresher to Single-cell Analysis with R and Bioconductor  <br> [Single-cell Analysis slides (PDF)](../slides/2023-01-31_b_scRNAseq_intro.pdf)    |
```

Comment (sjspielman) [reply workshop/SCHEDULE.md:12]: I sort of prefer this emphasis but don't feel too strongly, up to you!

**Review by sjspielman — APPROVED** [review]
👍 
I can't make this comment in the PR since it's not a diff, but can we add an `*Adjourn for the day*` to the end of Thursday? It can be a squishy time too, as long as we have something that indicates "presentations will not go on for 4 hours."

**Comment by allyhawkins** [comment]
Just a really minor and annoying comment but my name is the only one not in all caps in the intro slides...

**Comment by jashapiro** [comment]
> 👍 I can't make this comment in the PR since it's not a diff, but can we add an `*Adjourn for the day*` to the end of Thursday? It can be a squishy time too, as long as we have something that indicates "presentations will not go on for 4 hours."

It says *Adjourn* because the whole thing is over then...

**Review by allyhawkins — APPROVED** [review]
Actual slides and links look good other than the minor style difference in the instructor names, which I really don't feel strongly about so you can probably just leave it as is.

**Comment by jashapiro** [comment]
> Actual slides and links look good other than the minor style difference in the instructor names, which I really don't feel strongly about so you can probably just leave it as is.

Nope, fixed: https://github.com/AlexsLemonade/2023-january-training/pull/37/commits/ab49bd1c6f06f9e11dffc1276f44130fdf2ae7e7

**Pull request merged**